<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1470</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1531</width>
      <height>861</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1531</width>
       <height>805</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone 3 - Page 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>251</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>7</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 7pt &quot;Lucida Sans&quot;;
background-color: rgb(87, 114, 121);</string>
      </property>
      <property name="title">
       <string>Select Location</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget { 
background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;
}

QPushButton:hover {
    background-color: #0056b3;
}

QPushButton:pressed {
    background-color: #004080;
}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans&quot;;
</string>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans&quot;;
</string>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>231</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8 px;
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>0</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans&quot;;
</string>
       </property>
       <property name="text">
        <string>Zipcodes</string>
       </property>
      </widget>
      <widget class="QListWidget" name="zipcodeList">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>91</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Enter Business Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="business_name">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>180</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QListWidget" name="filterCategories">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>40</y>
         <width>251</width>
         <height>191</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <pointsize>7</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8 px;
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>0</y>
         <width>241</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans&quot;;
</string>
       </property>
       <property name="text">
        <string>Filter on Category:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="resetButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);
border: 2px solid gray;
border-radius: 10px;
font: 9pt;</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <zorder>label_18</zorder>
      <zorder>stateList</zorder>
      <zorder>label</zorder>
      <zorder>label_2</zorder>
      <zorder>cityList</zorder>
      <zorder>label_5</zorder>
      <zorder>zipcodeList</zorder>
      <zorder>label_6</zorder>
      <zorder>business_name</zorder>
      <zorder>filterCategories</zorder>
      <zorder>resetButton</zorder>
     </widget>
     <widget class="QGroupBox" name="businessGroup">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>1461</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Lucida Sans&quot;;
background-color: rgb(87, 114, 121);</string>
      </property>
      <property name="title">
       <string>Businesses</string>
      </property>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>1371</width>
         <height>271</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;
font: 9pt &quot;Lucida Sans&quot;;</string>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>0</y>
        <width>741</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>7</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 7pt &quot;Lucida Sans&quot;;
background-color: rgb(87, 114, 121);</string>
      </property>
      <property name="title">
       <string>Zipcode Statistics</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Lucida Sans&quot;;
</string>
       </property>
       <property name="text">
        <string>Number of 
Businesses</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Lucida Sans&quot;;
</string>
       </property>
       <property name="text">
        <string>Total 
Population</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Lucida Sans&quot;;
</string>
       </property>
       <property name="text">
        <string>Average 
Income</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTableWidget" name="categoryTable">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>40</y>
         <width>281</width>
         <height>191</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <pointsize>7</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>0</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans&quot;;
</string>
       </property>
       <property name="text">
        <string>Top Categories</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="income">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="businessNumber">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="population">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="businessGroup_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>580</y>
        <width>1471</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;;
background-color: rgb(87, 114, 121);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans&quot;;
</string>
       </property>
       <property name="text">
        <string>Popular Businesses (by zipcode)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>0</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans&quot;;
</string>
       </property>
       <property name="text">
        <string>Successful Businesses (by zipcode)</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="successful">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>30</y>
         <width>681</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lucida Sans</family>
         <pointsize>7</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;
font: 7pt;</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>5</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>5</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>5</number>
       </attribute>
      </widget>
      <widget class="QTableWidget" name="popular">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>651</width>
         <height>131</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(228, 228, 228);
border: 2px solid gray;
border-radius: 10px;
font: 7pt;</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1531</width>
       <height>805</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone 1 - Page 2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1021</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Lucida Sans&quot;;</string>
      </property>
      <property name="title">
       <string>Search Business by Name</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Enter Business Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>City for Business:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bname">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>40</y>
         <width>261</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="bcity">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>330</y>
         <width>261</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="businesses">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>261</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1470</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumilestone3">
    <property name="title">
     <string>milestone3</string>
    </property>
   </widget>
   <addaction name="menumilestone3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
